src/OpenTelemetry.Instrumentation.ElasticsearchClient/TracerProviderBuilderExtensions.cs

```diff
@@ -30,18 +32,53 @@ public static class TracerProviderBuilderExtensions
     /// Enables Elasticsearch client Instrumentation.
     /// </summary>
     /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+    /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+    public static TracerProviderBuilder AddElasticsearchClientInstrumentation(
+        this TracerProviderBuilder builder)
+    {
+        return AddElasticsearchClientInstrumentation(builder, name: null, configure: null);
+    }
+
+    /// <summary>
+    /// Enables Elasticsearch client Instrumentation.
+    /// </summary>
+    /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+    /// <param name="configure">Elasticsearch client configuration options.</param>
+    /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+    public static TracerProviderBuilder AddElasticsearchClientInstrumentation(
+        this TracerProviderBuilder builder,
+        Action<ElasticsearchClientInstrumentationOptions> configure)
+    {
+        return AddElasticsearchClientInstrumentation(builder, name: null, configure);
+    }
+
+    /// <summary>
+    /// Enables Elasticsearch client Instrumentation.
+    /// </summary>
+    /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+    /// <param name="name">Name which is used when retrieving options.</param>
     /// <param name="configure">Elasticsearch client configuration options.</param>
     /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
     public static TracerProviderBuilder AddElasticsearchClientInstrumentation(
         this TracerProviderBuilder builder,
-        Action<ElasticsearchClientInstrumentationOptions> configure = null)
+        string name,
+        Action<ElasticsearchClientInstrumentationOptions> configure)
```

Comment:
Please update changelog with following information
 - remove method with default attribute.
 - add new method parameter.

Reply:
Please update the CHANGELOG to indicate the newly added support for using options through name. You could refer to the CHANGELOG updates made in this PR for the same change:

https://github.com/open-telemetry/opentelemetry-dotnet/pull/3664/files#diff-4f436e3acc6e1645db5b52fff0eb65303bad34c84466f1461005dc0040f23232R8-R10